src/main/resources/schema/wildfly-client-ejb_3_0.xsd

```diff
@@ -67,6 +68,21 @@
         </xsd:attribute>
     </xsd:complexType>
 
+    <xsd:complexType name="tx-propagation-type">
+        <xsd:annotation>
+            <xsd:documentation>
+                Definition of an EJB transaction propagation.
```

Comment:
This is confusing.  How about "options to configure the behavior of transaction propagation"?

Reply:
Indeed. Will fix.